requirements/dev.txt

```diff
@@ -44,9 +44,9 @@ packaging==21.3
     #   tox
 pep517==0.13.0
     # via build
-pip-tools==6.9.0
+pip-tools==6.10.0
```

Comment:
No breaking change in new version https://github.com/jazzband/pip-tools/blob/main/CHANGELOG.md#6100-2022-11-13